### ISSUE TITLE
Test if email address is used to sign in to Docker hub?

### ISSUE THREAD

**Issue by hellonearthis** [docker/kitematic#2823]
Would it be with testing if an email address is used for the docker hub signin and poping up an error message.  I see weekly someone having issues with this. :panda_face:

**Comment by FrenchBen** [comment]
@hellonearthis we didn't use to allow signing via email and a PR was merged to do so. 
We could disable it for the time being, but it serves as a good reminder to get hub status on the issue.
If Hub allows you to login via email, it should also allow you to pull images with the email.

**Comment by hellonearthis** [comment]
Yes, having the Hub accept email or username would be the best solution.

**Comment by mhelmetag** [comment]
Email login was disabled in #3229 . I'm wondering if this still is an issue? Or what the issue is exactly...

**Comment by FrenchBen** [comment]
@mhelmetag this is why we had added login via email, since hub allowed it - Behind the scene there is a conversion of the username, which kitematic doesn't have. Disabling the login via email is ok for now. Ideally we would either swap out the user after login, or hub allows pulling via email.

**Comment by mhelmetag** [comment]
@FrenchBen , oh! That makes so much sense. Cool! Good to know what would be needed to re-implement the email login feature.
